src/super_gradients/training/datasets/segmentation_datasets/coco_segmentation.py

```diff
@@ -36,11 +36,15 @@ def __init__(self, dataset_classes_inclusion_tuples_list: list = None, *args, **
         kwargs['crop_size'] = dataset_hyper_params['crop_size'] if 'crop_size' in dataset_hyper_params.keys() else 512
         # FIXME - Rescale before RandomRescale is kept for legacy support, consider removing it like most implementation
         #  papers regimes.
-        kwargs["image_mask_transforms_aug"] = transform.Compose([RandomFlip(),
-                                                                 Rescale(long_size=kwargs["img_size"]),
-                                                                 RandomRescale(scales=(0.5, 2.0)),
-                                                                 PadShortToCropSize(crop_size=kwargs['crop_size']),
-                                                                 CropImageAndMask(crop_size=kwargs['crop_size'], mode="random")])
+        default_image_mask_transforms_aug = transform.Compose([RandomFlip(),
+                                                               Rescale(long_size=kwargs["img_size"]),
+                                                               RandomRescale(scales=(0.5, 2.0)),
+                                                               PadShortToCropSize(crop_size=kwargs['crop_size']),
+                                                               CropImageAndMask(crop_size=kwargs['crop_size'], mode="random")])
+        kwargs["image_mask_transforms_aug"] = dataset_hyper_params.get("image_mask_transforms_aug",
```

Comment:
Does `get()` work on params from yamls?

Reply:
`get(...)` works on configs from `hydra.utils.instantiate` , but doesn't work on `HpmStruct`. @shaydeci do we need to support `dataset_hyper_params`  stored in `HpmStruct`?